### PR TITLE
SmartSense Open Closed Sensor: DevWS submission for INGENIUM ZB Magnetic ON/OFF Sensor

### PR DESCRIPTION
**Pull request by Duncankn**

### CODE REVIEW

**Comment by shrakuma** [comment]
@MAblewiczS can your team please review this DTH . there is a FP change and a change also to the DTH code
cc @tpmanley

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 1]
```diff
@@ -38,14 +38,16 @@ metadata {
...
+        fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "Megaman", model: "MS601/z1", deviceJoinName: "INGENIUM ZB Magnetic ON/OFF Sensor"
 	}
 
 	simulator {
 
 	}
 
 	preferences {
-		input "tempOffset", "number", title: "Temperature Offset", description: "Adjust temperature by this many degrees", range: "*..*", displayDuringSetup: false
+		input title: "Temperature Offset", description: "This feature allows you to correct any temperature variations by selecting an offset. Ex: If your sensor consistently reports a temp that's 5 degrees too warm, you'd enter \"-5\". If 3 degrees too cold, enter \"+3\".", displayDuringSetup: false, type: "paragraph", element: "paragraph"
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: Please rebase the changes on the latest `master` branch and undo this change. We recently changed this preferences code and it should be left as is.

Comment (shrakuma) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: @Duncankn  can you please take a look at this

Comment (Duncankn) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: @tpmanley the code was rebased, please check, thanks!

**Review by tpmanley — APPROVED** [review]

**Comment by tpmanley** [comment]
Changes look good but I'm holding on merging until someone from the CST side gives us the go ahead

**Comment by Duncankn** [comment]
Hi @tpmanley ,

Any update on this PR?
Thanks!

**Comment by tpmanley** [comment]
@Duncankn sorry about that, merged now

**Comment by shrakuma** [comment]
@Duncankn I accidentally rejected this in developer workspace. the rejection in dev ws is just cosmetic.  Please note that this change has made it to our platform on our end.

**Comment by shrakuma** [comment]
also what's the name that you have provided for the instructions set for this device? 
the device has instructions 
"Short press the reset button to enable pairing, The LED indicator flashes once for a second. If the sensor is no flashing, long-pressing the reset button to reset the sensor and the indicator LED will flash quickly."

**Comment by Duncankn** [comment]
> also what's the name that you have provided for the instructions set for this device?
> the device has instructions
> "Short press the reset button to enable pairing, The LED indicator flashes once for a second. If the sensor is no flashing, long-pressing the reset button to reset the sensor and the indicator LED will flash quickly."

Hi @shrakuma 
Sorry I missed the message!
The instruction name is "INGENIUM ZB Magnetic On/Off Sensor".
